wagtail/project_template/home/templates/home/welcome_page.html

```diff
@@ -0,0 +1,377 @@
+{% templatetag openblock %} load i18n {% templatetag closeblock %}
+<link rel="stylesheet" type="text/css" href="/static/admin/css/fonts.css">
+<style type="text/css">
+  body, main {
+    margin: 0 auto;
+  }
+  .body, .tip {
+    stroke: #fff;
+  }
+  html {
+    line-height: 1.15;
+    box-sizing: border-box;
```

Comment:
I think this should be used on all elements on the page instead of just a few. Something like:

```scss
*,
*::before,
*::after {
    box-sizing: inherit;
}

body {
    box-sizing: border-box;
}
```

Reply:
I updated this to use the latest recommendation from [Paul Irish](https://www.paulirish.com/2012/box-sizing-border-box-ftw/).